.github/workflows/build.yml

```diff
@@ -16,6 +16,10 @@ jobs:
         ports:
           - 6379:6379
         options: --health-cmd="redis-cli ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      nacos:
+        image: nacos/nacos-server:2.4.2
```

Comment:
这个镜像貌似拉不到，建议换一个，然后你要看下鉴权这块，是否需要配置下否则默认nacos如果开启了鉴权，ci也是过不了的，要保证关闭接口鉴权
This mirror doesn't seem to be able to be pulled. It is recommended to change it. Then you have to take a look at the authentication piece and see if it needs to be configured. Otherwise, if the authentication is turned on by nacos by default, ci will not be able to pass. Make sure to turn off the interface authentication.

Reply:
好的，关闭了鉴权和增加了健康检测，等这次ci流程结果看看

Reply:
> 好的，关闭了鉴权和增加了健康检测，等这次ci流程结果看看

好像这么改动后，build的ci直接没了，不知道是不是yaml读取出现了问题，比如某些参数填写是错的，导致失效了，帮忙看下哈。
It seems that after making this change, the build CI failed, and I’m not sure if it’s due to issues with reading the YAML file, such as incorrect parameters causing it to become invalid. Could you please help check this?

Reply:
好的，原本以为是docker compose的写法，刚刚纠正为‘GitHub Actions’的yaml格式了，再试试看

Reply:
> 好的，原本以为是docker compose的写法，刚刚纠正为‘GitHub Actions’的yaml格式了，再试试看

好像不行，是不是github actions容器参数设置的写法不太一样？
It doesn't seem to work. Is the github actions container parameter setting different?